### PR TITLE
Do not always load metadata on webui browse

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java [hunk 1]
```diff
@@ -459,7 +459,7 @@ public Response getWebUIBrowse(@DefaultValue("/") @QueryParam("path") String req
         }
 
         filesInfo = mFileSystemMaster.listStatus(currentPath, ListStatusContext.mergeFrom(
-            ListStatusPOptions.newBuilder().setLoadMetadataType(LoadMetadataPType.ALWAYS)));
+            ListStatusPOptions.newBuilder()));
```
Comment (yuzhu) [inline core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java:462]: maybe use `ListStatusContext.defaults()` ?

**Review by yuzhu — COMMENTED** [review]
@bradyoo Minor nit. 
Ideally we should be using the stream based listStatus call here since we paginate on the web Ui side as well. Otherwise the web ui will wait forever for the listStatus to fetch all the directories. but we can address that in a separate PR.

**Comment by yuzhu** [comment]
is the test failure relevant?

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Comment by bradyoo** [comment]
@yuzhu Should we cherry-pick this? if so, how far back?

**Pull request merged**